client/src/components/cityview/realm/villagers/NpcListComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import { useDojo } from "../../../../DojoContext";
+// import { Realm } from "../../../../types";
+import Avatar from "../../../../elements/Avatar"
+import Button from "../../../../elements/Button";
+import { useState } from "react";
+import { NpcPopup } from "./NpcPopup";
+
+
+type npcProps = {
+    id: number;
+    name: string;
+    image: string;
+    age: number;
+    sex: string;
+    role: string;
+    hungriness: string;
+    happiness: string;
+    belligerent: string;
+    description: string;
+}
+
+type NpcComponentProps = {
+    // npcId: number;
+    npc: npcProps;
+    // realm: Realm;
+
+};
+
+export const NpcListComponent = ({npc}: NpcComponentProps) => {
+    const {
+        // setup: {
+            // components: {},
+            // systemCalls: {},
+            // optimisticSystemCalls: {},
+        // },
+        account: { account },
+    } = useDojo();
+
+    const [showNpcStats, setShowNpcStats] = useState(false);
+
+    return (
+        <div className="relative flex flex-col border rounded-md border-gray-gold text-xxs text-gray-gold">
+            
+            {npc && (<>
+                
+                <div className="flex flex-row">
+                    <div className="flex flex-col mr-10">
+
+                    
+                        <Button
+                            className="object-left-top !px-[6px] !py-[2px] text-xxs border-gray-gold !text-gray-gold text-white/70 self-start"
+                            variant="success"
+                            // disabled={nextHarvest === 0}
```

Comment:
Can all these commented lines be removed?

Reply:
Done